protos/connection_common.proto

```diff
@@ -52,6 +52,12 @@ message ServiceException {
   repeated ServiceException child_exception = 7;
 }
 
+message Notification {
+  string connection_id = 1;
+  int32 notification_id = 2;  // TODO check if this is a long <-> int64
```

Comment:
a forgotten todo?

Reply:
Yep, it was a TODO left by Guido. I checked the wsdl, notification id is a xsd:long and to accommodate this I defined it it as a int64.